src/screen/AllUser.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  FlatList,
+  TouchableOpacity,
+} from "react-native";
+import FontAwesome from "react-native-vector-icons/FontAwesome5";
+import { connect } from "react-redux";
+import Constants from "expo-constants";
+import { GetUser } from "../redux/action/action";
+
+const AllUser = (props) => {
+  console.log(props.data, "dataaaaaaaa");
```

Comment:
make habit of removing console. i will not remind again and again. really frustrating